tests/e2e/test_spyre_cb.py

```diff
@@ -44,6 +45,64 @@ def test_cb_max_tokens(
                                    monkeypatch=monkeypatch)
 
 
+@pytest.mark.cb
+@pytest.mark.parametrize("cb", [True])
```

Comment:
Can we also add a test for static batching? Because the logic for that is actually more complicated because the max context length is conditioned on the eligible warmup shapes.